src/GLideNUI/configDialog.ui

```diff
@@ -208,6 +208,16 @@
             </widget>
            </item>
            <item>
+            <widget class="QCheckBox" name="vSyncCheckBox">
+             <property name="toolTip">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vertical sync, or VSync, can improve the image by syncing the game's frame rate to your monitor's refresh rate. This can prevent image tearing, but may cause emulation problems.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;[Recommended: &lt;span style=&quot; font-style:italic;&quot;&gt;Usually off, on if you have no problems&lt;/span&gt;]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Too much space between tooltip message and [Recommended:
![vsync_tooltip](https://cloud.githubusercontent.com/assets/8983092/23110202/0b72be3c-f752-11e6-94b6-123b711f5917.png)